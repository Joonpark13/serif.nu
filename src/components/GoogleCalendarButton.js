import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import IconButton from '@material-ui/core/IconButton';
import EventIcon from '@material-ui/icons/Event';
import Tooltip from '@material-ui/core/Tooltip';


const useStyles = makeStyles({
  icon: {
    color: 'white',
  },
});

export default function GoogleCalendarButton() {
  const classes = useStyles();
  return (
    <div>
      <Tooltip title="Export to Google Calendar">
        <IconButton aria-label="GoogleCalendar">
          <EventIcon className={classes.icon} />
        </IconButton>
      </Tooltip>
    </div>
  );
}
