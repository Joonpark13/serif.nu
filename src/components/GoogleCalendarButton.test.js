import React from 'react';
import { shallow } from 'enzyme';
import GoogleCalendarButton from './GoogleCalendarButton';

describe('GoogleCalendarButton', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<GoogleCalendarButton menuAction={() => {}} />);

    expect(wrapper.get(0)).toMatchSnapshot();
  });
});
