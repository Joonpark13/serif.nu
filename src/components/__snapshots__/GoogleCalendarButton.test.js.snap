// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleCalendarButton renders correctly 1`] = `
<div>
  <ForwardRef(WithStyles)
    title="Export to Google Calendar"
  >
    <ForwardRef(WithStyles)
      aria-label="GoogleCalendar"
    >
      <pure(Event)
        className="makeStyles-icon-1"
      />
    </ForwardRef(WithStyles)>
  </ForwardRef(WithStyles)>
</div>
`;
